internal/index/indexoperations/index.go

```diff
@@ -42,8 +42,11 @@ func ListIndexes(paths environment.Paths) ([]Index, error) {
 	}
 
 	indexes := []Index{}
-	for _, dir := range dirs {
-		indexName := dir.Name()
+	for _, file := range dirs {
```

Comment:
It is confusing that `dirs` does not contain directories but also files. Let's make this clear by renaming to `entries`.

```suggestion
	for _, e := range entries {
```

Reply:
Thanks, will work on it now :)